spec/rex/powershell/output_spec.rb

```diff
@@ -36,11 +36,9 @@
   end
 
   describe "::deflate_code" do
-    it 'should zlib the code and wrap in powershell in uncompression stub' do
+    it 'should zlib the code and wrap in powershell in decompression stub' do
       compressed = subject.deflate_code
       expect(compressed.include?('IO.Compression.DeflateStream')).to be_truthy
-      compressed =~ /FromBase64String\('([A-Za-z0-9\/+=]+)'\)/
-      expect($1.size).to be < Rex::Text.encode_base64(example_script).size
```

Comment:
Removed this because the assumption that the base64'ed contents exist as a string literal is no longer accurate.